win_etw_macros/src/lib.rs

```diff
@@ -237,6 +237,11 @@ fn trace_logging_events_core(attr: TokenStream, item_tokens: TokenStream) -> Tok
     // Definitions that go inside the "impl MyProvider { ... }" block.
     let mut provider_impl_items = TokenStream::new();
 
+    // To make this simple, we either require all events to be labeled
+    // with an event id or autogenerated.
+    let mut auto_generated = true;
```

Comment:
Please rename `auto_generated` to `ids_are_auto_generated` or some such, for clarity.